rho/postprocessing.py

```diff
@@ -229,9 +235,11 @@ def empty_output_message(val, name):
     if JBOSS_EAP_INSTALLED_VERSIONS in fact_names:
         val[JBOSS_EAP_INSTALLED_VERSIONS] = (
             empty_output_message('; '.join(jboss_releases), 'jboss'))
+        val[JBOSS_EAP_INSTALLED_VERSIONS + MR] = jboss_releases
     if JBOSS_EAP_DEPLOY_DATES in fact_names:
         val[JBOSS_EAP_DEPLOY_DATES] = (
             empty_output_message('; '.join(deploy_dates), 'jboss'))
+        val[JBOSS_EAP_DEPLOY_DATES + MR] = deploy_dates
```

Comment:
I think the deploy dates may be useful in the report as they may help for when a subscription was needed.

Reply:
Same as previous comment.